docs/en/core/writing-modes.md

```diff
@@ -0,0 +1,16 @@
+# Writing Modes
+In the View Menu, Zettlr features several Editor settings that may enhance your writing experience. Have a play and see what works for you!
```

Comment:
Make sure to add an empty line in between headings and the text

Reply:
Fixed